pkg/ccl/changefeedccl/schemafeed/schema_feed.go

```diff
@@ -58,12 +58,26 @@ func (e TableEvent) Timestamp() hlc.Timestamp {
 // leaseAcquirer is an interface containing the methods on *lease.Manager used
 // by the schema feed.
 type leaseAcquirer interface {
-	Acquire(ctx context.Context, timestamp hlc.Timestamp, id descpb.ID) (lease.LeasedDescriptor, error)
+	Acquire(ctx context.Context, timestamp lease.ReadTimestamp, id descpb.ID) (lease.LeasedDescriptor, error)
 	AcquireFreshestFromStore(ctx context.Context, id descpb.ID) error
 	// TODO(yang): Investigate whether the codec can be stored in the schema feed itself.
 	Codec() keys.SQLCodec
 }
 
+// leaseTS is a wrapper around hlc.Timestamp which implements the lease.ReadTimestamp
+// interface.
+type leaseTS hlc.Timestamp
```

Comment:
Could you use `lease.TimestampToReadTimestamp` here too instead of creating a new wrapper?